### PR TITLE
CHEF-9848 - Exclude Warning "Unrecognized feature name"  for Custom InSpec plugins.

### PR DESCRIPTION
**Pull request by Vasu1105**
## Description

This PR is raised to exclude warning  "Unrecognized feature name"  for Custom plugins.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: test/unit/feature_test.rb [hunk 1]
```diff
@@ -62,6 +79,17 @@
       _(logger_io.string).must_match(/WARN/)
       _(logger_io.string).must_match(/test-feature-nonesuch/)
     end
+
+    it "should not give warnings for inspec custom plugins" do
```
Comment (ahasunos) [inline test/unit/feature_test.rb:83]: Could you add another test where it gives warning, executing line 17 of runner.rb?

Comment (ahasunos) [reply test/unit/feature_test.rb:83]: Ignore, if we have such a test already.

Comment (Vasu1105) [reply test/unit/feature_test.rb:83]: Yes it's already there. https://github.com/inspec/inspec/pull/6988/files#diff-2c333ccd448d35de740f548d16f093e60c52c9036914b72ce0ecbb654b934f2bR79

Comment (Vasu1105) [reply test/unit/feature_test.rb:83]: Ref: https://github.com/inspec/inspec/pull/6988/files#diff-2c333ccd448d35de740f548d16f093e60c52c9036914b72ce0ecbb654b934f2bR79

**Review by ahasunos — APPROVED** [review]

**Review by Nik08 — APPROVED** [review]

**Pull request merged**